### PR TITLE
Project

### PR DESCRIPTION
**Pull request by Mehdi-Rh**
This pull request is about the whole capstone project.
Requirement  of the project: 

- You should personalize the content of your page. Choose a topic that is different than the one in the original design.
- You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:
- Colors.
- Typography: font face, size and weight.
- Layout: composition and space between elements.
- The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- You can use a CSS framework (for example Bootstrap) for styling, if you want to
- You should build only these 2 pages:
- The home page.
- The about page.
- Each of these pages should have versions for 2 different screen sizes:
- Mobile: up to 768px wide.
- Desktop: 768px or wider.
- Interactions
- Links
- The home page should have a link in the menu to the about page.
- The logo in the header links to the home page.
- Mobile menu
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc.).
- The mobile menu should have a close (X) button that closes the menu.
- Dynamic page
- 
- The section "Featured speakers" should be created dynamically in JavaScript.
- You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+        
+        If you want to come along with me to improve our skills throughout different projects and problems that I face in my job.
+        We will learn how to approach and solve problems, how to debug code and find the source of the problem…
+        
+        If you are a new coder or someone who wants to keep learning, than this is the place for you!
+       </p></div>
+      <p>
+        Please contact us if you have any question:<br><b>sample-email@sample.mail</b>
+      </p>
+    </section>
```
Comment (VICTOR-CHUKWU) [inline about.html:63]: - [ ] @Mehdi-Rh , great job for such a wonderful design, you have just a little change to make to the design to give an inch-perfect work. kindly make your `<section id="hero">` to have a dark background and the paragraph that contains the text `Hello and welcome to the blog!` to have a white background just as it looks in the design
     your design 👇 
                       image

   Figma design 👇 
             
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <section id="hero">
+      <h2>"Hello World! We Are Algerian Coders</h2>
```
Comment (VICTOR-CHUKWU) [inline about.html:49]: - [ ] [Optional] Following the Figma design, Kindly hide this in desktop view as it does not show in the design desktop view, you can do that with media query

File: index.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+            <li><p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Illum, harum!</p></li>
+          </ul>
+        </div>
+        <div class="program-card">
+          <ul>
+            <li><img src="images/program-icon/icons-5.svg" alt="Program Icon"></li>
+            <li><h4><b>AC 2022</b></h4></li>
+            <li><p>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Illum, harum!</p></li>
+          </ul>
+        </div>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [ ] From the design the cards are supposed to be five in number, kindly fix this
              design 👇 
             
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @Mehdi-Rh , 

Good job, your design is almost perfect and I like how organized your code base looks. keep it up 👍 .
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!


## Suggested changes
- [ ] Kindly check your Live demo link as it is not showing any content
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by MrBrN197 — APPROVED** [review]
## Hi, @Mehdi-Rh  👋

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**